eos-paygd/eos-paygd-2.service.in

```diff
@@ -9,9 +9,13 @@ ConditionPathExists=/etc/initrd-release
 ConditionKernelCommandLine=eospayg
 # Don't complete boot without this running
 FailureAction=poweroff
-# Start early to reduce the chance the user is dropped to a shell before we've started
-After=sysinit.target
-Before=basic.target
+# We used to start earlier in an effort to reduce the chance the user is dropped to a
+# shell before we started - but that leaves us with a weird race condition that can
+# sometimes lead to chroot()ing to a place where dbus doesn't exist.
+#
+# We'll have to solve the root shell problem elsewhere.
+After=initrd-root-fs.target
+Before=dracut-pre-pivot.service
```

Comment:
Should this be a target as well (ex.: initrd.target), instead of dracut-pre-pivot.service?

Also, do we need to be after any of the mounts performed by ostree (via the ostree generators)?

Reply:
I had thought initrd-root-fs.target got us after those generators, which come before local-fs.target?

Looking at `man bootup` again, I missed something first time around - I think we have to start before initrd-parse-etc.service to protect against x-initrd.mount options in fstab.

I've force pushed that change.